### DISCUSSION TITLE
Question regarding virtual chinrest plugin method of resizing

### DISCUSSION THREAD

**Discussion by yuvalharr** [jspsych/jsPsych#1635] [Q&A] [answered]
Hey all,

First of all, I was delighted to see that the chinrest plugin was already implemented into the new Jspsych release! Thanks very much for anyone who worked on that!

I Have a question regarding the degree resize option. Hoping that anyone here could help me with that:

When using resize_units = "deg", does it mean that in all screen sizes and distances, one degree would be equal to a **fixed objective length**? Or, does it only guarantee me that one degree would be equal to a **fixed number of pixels**?

If the latter is correct, then I guess that screens with different sizes/resolutions will lead to a different presented size on them (after resize), even if participants are sitting in exactly the same distance from the screen. If so, is there any way of resizing so that 1 degree would be equal to a fixed length (e.g., cm) instead of pixels?

Thanks for any help!

Yuval

**Comment by jodeleeuw** [comment, marked as answer]
Hi Yuval,

It's actually a third option: one degree (a fixed objective length) is set to be equal to a non-transformed pixel value. So if one degree equals 50 pixels and you have an image that is 200 pixels wide, then the screen will be resized so that your 200 pixel image takes up the correct number of pixels to be 4 degrees wide.

**Reply by yuvalharr** [reply]
Great, now I get it! 

Thanks very much!

**Reply by pjkohler** [reply]
Note that resizing is optional, the plugin can also just compute the conversion between pixels and physical units w/o any resizing. Glad you like the plugin.
